src/gleam/erlang/port.gleam

```diff
@@ -10,18 +7,3 @@ import gleam/dynamic/decode.{type DecodeError}
 /// [1]: https://erlang.org/doc/reference_manual/ports.html
 ///
 pub type Port
```

Comment:
I was left confused after reading the documentation; the doc is talking about sending/reading bytes from a port but there's no functionality to do any of that, does it make sense to have an empty module with just this external type?

Reply:
This library is the foundation of Erlang based code in the Gleam world, so it defines some types and functions that other programs will build on. It's like how you can't actually do anything with just `gleam_http` or the promise module from `gleam_javascript`, but the other packages can interop via it.

Reply:
Aah I see, thanks for the explanation!